### PR TITLE
Add new resource: Migration Policy

### PR DESCRIPTION
**Pull request by dshchedr**

### CODE REVIEW

File: ocp_resources/migration_policy.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    """
+    MigrationPolicy object
+    """
+
+    api_group = Resource.ApiGroup.MIGRATIONS_KUBEVIRT_IO
+
+    def __init__(
+        self,
+        name=None,
+        allow_auto_converge=None,
```
Comment (myakove) [inline ocp_resources/migration_policy.py:14]: Please add a docstring describe the args

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/migration_policy.py (hunk 1 above)
Comment (dshchedr) [reply ocp_resources/migration_policy.py:11]: added

File: ocp_resources/migration_policy.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        """
+        Create MigrationPolicy object.
+
+        Args:
+            name (str): Migration Policy name
+            allow_auto_converge (bool, optional)
+            allow_post_copy (bool, optional)
+            bandwidth_per_migration (str, optional, i.e. "1Gi")
+            completion_timeout_per_gb (int, optional)
+            namespace_selector (dict, optional): Dict of Namespace labels selectors
```
Comment (myakove) [inline ocp_resources/migration_policy.py:28]: `namespace_selector` and `vmi_selector`
I think we need some example for the dict

File: ocp_resources/migration_policy.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        if self.allow_auto_converge:
+            res["spec"]["allowAutoConverge"] = self.allow_auto_converge
+        if self.bandwidth_per_migration:
+            res["spec"]["bandwidthPerMigration"] = self.bandwidth_per_migration
+        if self.completion_timeout_per_gb:
+            res["spec"]["completionTimeoutPerGiB"] = self.completion_timeout_per_gb
+        if self.allow_post_copy:
+            res["spec"]["allowPostCopy"] = self.allow_post_copy
+
+        for key, value in self.namespace_selector.items():
```
Comment (myakove) [inline ocp_resources/migration_policy.py:58]: I think we should get the dict as it should be and set it, no need to iter on it.

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/migration_policy.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+class MigrationPolicy(Resource):
+
+    api_group = Resource.ApiGroup.MIGRATIONS_KUBEVIRT_IO
+
+    def __init__(
+        self,
+        name=None,
+        allow_auto_converge=None,
+        allow_post_copy=None,
```
Comment (rnetser) [inline ocp_resources/migration_policy.py:12]: All bool args should have a bool default value

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/migration_policy.py (hunk 3 above)
Comment (dshchedr) [reply ocp_resources/migration_policy.py]: right, thanks!

File: ocp_resources/migration_policy.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dshchedr) [reply ocp_resources/migration_policy.py]: updated with example

**Review by myakove — DISMISSED** [review]

**Comment by myakove** [comment]
@dshchedr Please rebase

**Comment by dshchedr** [comment]
> @dshchedr Please rebase

done

**Review by myakove — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Comment by myakove** [comment]
@dshchedr please verify

**Comment by dshchedr** [comment]
!verified

**Comment by myakove** [comment]
!verified

**Pull request merged**